plugin-server/src/utils/db/db.ts

```diff
@@ -586,6 +571,33 @@ export class DB {
         return client ? kafkaMessages : updatedPerson
     }
 
+    public async updatePersonPropertiesPooled(
```

Comment:
why pooled? because of the client? 

do we care about this in the code?

it seems confusing to me as it kind of implies we batch updates. reading the name i'm expecting us to enqueue updates and only then run them

Reply:
We use it in a transaction, which is similarly to `addDistinctIdPooled`